.github/workflows/install-testing.yml

```diff
@@ -0,0 +1,86 @@
+name: Installation Tests
+
+on:
+  push:
```

Comment:
Anytime a pull request or push event happens and the workflow files are changed, we should run the tests to confirm that everything works as expected after the changes.